libs/tup-components/src/accounts/ManageAccount.module.css

```diff
@@ -50,6 +50,10 @@
     width: 200px;
 }
 
+.tap-href {
+    width: fit-content;
+}
```

Comment:
Excellent.

_I hope this won't be needed once `<Button>` and `<a>` are not used together, but a great solution to the current problem._